content/technotes/2020-05-28-rsvg2/index.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: rsvg2
+title: 'Superior svg graphics rendering in R, and why it matters'
+date: '2020-05-28'
+author:
+  - Jeroen Ooms
+cover: https://docs.ropensci.org/av/reference/read_audio-1.png
+tags:
+  - R
+  - images
+  - packages
+  - svg
+---
+
+This week we released a major new version of the [rsvg](https://cran.r-project.org/web/packages/rsvg/index.html) package on CRAN. This package provides R bindings to [librsvg2](https://developer.gnome.org/rsvg/stable/) which is a powerful system library for rendering svg images into bitmaps, which we can use for further processing in for example the [magick](https://docs.ropensci.org/magick) package.
+
+The biggest change in this release is the R package on Windows and MacOS now includes the latest librsvg 2.48.4. This is a major upgrade; the librsvg2 rendering engine has been [completely rewritten](https://people.gnome.org/~federico/blog/css-in-librsvg-is-now-in-rust.html) in Rust using components from [Mozilla Servo](https://research.mozilla.org/servo-engines/). This has resulted in major improvements in quality and performance, and we have gained full support for css styling.
```

Comment:
I know of Rust thanks to your talk, could you add a footnote about it? https://www.youtube.com/watch?v=MK9YNr3FTQs

Reply:
I don't know, maybe I'll refer to the slides instead 🙈

Reply:
you can refer to both, _you_ don't have to watch the talk :wink: